frontend/lib/about/dev_settings_view.dart

```diff
@@ -123,7 +125,7 @@ class DevSettingsView extends StatelessWidget {
   }
 
   Future<void> _setSampleCard(BuildContext context) async {
-    Provider.of<UserCodeModel>(context, listen: false).setCode(_determineUserCode(buildConfig.projectId.local));
+    Provider.of<UserCodeModel>(context, listen: false).insertCode(_determineUserCode(buildConfig.projectId.local));
```

Comment:
I don't think we should allow seting more than allowed cards. Even through dev settings, right? :)